src/Resources/translations/SonataMediaBundle.en.xliff

```diff
@@ -14,6 +14,10 @@
         <source>gallery</source>
         <target>Gallery</target>
       </trans-unit>
+      <trans-unit id="gallery_item">
+        <source>gallery_item</source>
+        <target>Gallery item</target>
```

Comment:
We have the following translation
```
<trans-unit id="form.label_gallery_items">
        <source>form.label_gallery_items</source>
        <target>Media</target>
      </trans-unit>
```
Should we use Media here then ?

Reply:
Do you want to translate gallery_item into "Media", that is the same translation of media itself? it would be confusing in sonata user admin then.

Reply:
I don't use this bundle and don't know the difference.
I just notice that if gallery_item is different from media, maybe the translation of `form.label_gallery_items` is wrong.

I think @jordisala1991 use this lib.

Reply:
That key is actually used in the gallery admin (it's the **Media*** label):

![Immagine 2023-03-10 133152](https://user-images.githubusercontent.com/1532616/224316747-8dfc6639-94a3-416e-b40d-48a6013f2da3.png)